CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## v1.2.0 (04/02/2020)
+
+**Common**
+
+* Updated the development dependencies.
+
+**olp-sdk-dataservice-read**
+
+* Added the `IndexLayerClient` class that is used to access index layers on OLP. This class implements the `getPartitions` and `getData` methods.
+* Added the `HttpError` class that extending Error class and adding status code to the http errors.
+* Improved errors propagations in all public methods. Now the methods rejects the Promises with instance of `HttpError` or `Error` class in case of error.
+
+**olp-sdk-authentication**
+
+* Added the `HttpError` class that extending Error class and adding status code to the http errors.
+* Improved errors propagations in all public methods. Now the methods rejects the Promises with instance of `HttpError` or `Error` class in case of error.
+
+* **Breaking Change** Improved return type for the `getEarliestVersion` method in `CatalogClient`. Return data type changed to be the same as `getLatestVersion` method.
+* **Breaking Change** Public methods rejects the Promises with instance of `HttpError` or `Error` class instead of strings in case of error.
```

Comment:
```suggestion
* **Breaking Change** Public methods now reject the promises with the `HttpError` instance or `Error` class instead of strings when errors occur.
```